### ISSUE TITLE
致命错误：bad revision 'HEAD'

### ISSUE THREAD

**Issue by wu-xiaochen** [xorbitsai/inference#1540]
Note that the issue tracker is NOT the place for general support.
I have used “xinference-local --host 0.0.0.0 --port 9997” to start Xinference server, but it shows "致命错误：bad revision 'HEAD'".And the shell stoped at "Starting Xinference at endpoint: http://0.0.0.0:9997"

**Comment by qinxuye** [comment]
Can you open the UI, normally the command line will stop at `starting xxx`.

**Comment by wu-xiaochen** [comment]
> Can you open the UI, normally the command line will stop at `starting xxx`.

yes,i can open the UI and fastAPI pages,  but when i want to test the api, it shows error~

**Comment by WSKNOO** [comment]
same question，so how to fix it?

**Comment by wu-xiaochen** [comment]
i did not fix it. The xinference works well for a moment
